packages/components/src/DataList/DataList.css

```diff
@@ -1,7 +1,15 @@
 .wrapper {
   flex: 1;
 }
+.titleContainer {
+  display: flex;
+  align-items: center;
+}
 
+.results {
+  width: 80px;
```

Comment:
This would be better as min-width because this would happen

![image](https://github.com/GetJobber/atlantis/assets/15986172/5b4a54df-004a-4b28-a3f7-4610b47093d0)

which should make the glimmer still work

```suggestion
  min-width: 80px;
```